packages/itwin/tree-widget/src/components/trees/common/components/TreeNodeCheckbox.tsx

```diff
@@ -40,19 +40,21 @@ export function TreeNodeCheckbox({ node, onCheckboxClicked, getCheckboxState, ..
 
   const checkboxState = getCheckboxState(node);
   return (
-    <Checkbox
-      {...props}
-      className={cx("tw-tree-node-checkbox", props.className)}
-      checked={checkboxState.state === "on"}
-      onClick={(e) => {
-        e.stopPropagation();
-      }}
-      onChange={(e) => {
-        onCheckboxClicked(node, e.currentTarget.checked);
-      }}
-      indeterminate={checkboxState.state === "partial"}
-      disabled={checkboxState.isDisabled}
-      title={checkboxState.tooltip}
-    />
+    <Tooltip content={checkboxState.tooltip} placement="left">
+      <Checkbox
+        {...props}
+        className={cx("tw-tree-node-checkbox", props.className)}
+        checked={checkboxState.state === "on"}
+        onClick={(e) => {
+          e.stopPropagation();
+        }}
+        onChange={(e) => {
+          onCheckboxClicked(node, e.currentTarget.checked);
+        }}
+        indeterminate={checkboxState.state === "partial"}
+        disabled={checkboxState.isDisabled}
+        title={checkboxState.tooltip}
```

Comment:
I think `title` should be removed to avoid double tooltips.

Reply:
I guess the tooltip wrapper adds title anyway, removed